Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -1534,13 +1534,24 @@ static Neighborhood retrieve_neighborhood(Channel const& channel, u32 x, u32 y)
     return neighborhood;
 }
 
+struct ModularOption {
```

Comment:
nit: ModularOptions

Reply:
Thanks, also fixed it for `PassGroupOptions`.